tests/lit-tests/export-c.ispc

```diff
@@ -0,0 +1,7 @@
+// RUN: %{ispc} --target=host --emit-asm %s -o - 2>&1 | FileCheck %s
+
+// CHECK-NOT: foo___:
+// CHECK-NOT: Error: syntax error, unexpected "C".
+// CHECK: foo:
+
+export "C" void foo() { return; }
```

Comment:
I would also check that `export "C"` function is present in generated header file.

Reply:
Done